ui/plugins/auto_classification/controller.js

```diff
@@ -976,7 +976,7 @@ treeherder.controller('ClassificationPluginCtrl', [
         };
 
         $scope.canIgnore = function() {
-            return $scope.pendingLines().length > 0;
+            return $scope.pendingLines().length > 0 && $scope.loggedIn();
```

Comment:
Actually now that I look at it, this change isn't necessary. It's probably better to just restrict this at the html/template level, as you are.